gene/query.py

```diff
@@ -94,10 +94,11 @@ def add_record(self,
             containing name of the source of the match
         """
         del item['label_and_type']
-        label_types = ['aliases', 'other_identifiers', 'previous_symbols']
-        for label_type in label_types:
-            if label_type not in item.keys():
-                item[label_type] = []
+        attr_types = ['aliases', 'other_identifiers', 'previous_symbols',
+                       'xrefs']
```

Comment:
Whoops -- I set you up for a flake8 error there:

```suggestion
        attr_types = ['aliases', 'other_identifiers', 'previous_symbols',
                      'xrefs']
```